src/helpers/pivot/pivot_runtime_definition.ts

```diff
@@ -36,33 +36,37 @@ export class PivotRuntimeDefinition {
     return dimension;
   }
 
-  getMeasure(name: string): PivotMeasure {
-    const measure = this.measures.find((measure) => measure.name === name);
+  getMeasure(id: string): PivotMeasure {
+    const measure = this.measures.find((measure) => measure.id === id);
     if (!measure) {
-      throw new EvaluationError(_t("Field %s is not a measure", name));
+      throw new EvaluationError(_t("Field %s is not a measure", id));
     }
     return measure;
   }
 }
 
 function createMeasure(fields: PivotFields, measure: PivotCoreMeasure): PivotMeasure {
-  const name = measure.name;
+  const fieldName = measure.fieldName;
   const field =
-    name === "__count"
+    fieldName === "__count"
       ? { name: "__count", string: _t("Count"), type: "integer", aggregator: "sum" }
-      : fields[name];
+      : fields[fieldName];
   const aggregator = measure.aggregator;
   return {
-    nameWithAggregator: name + (aggregator ? `:${aggregator}` : ""),
+    /**
+     * Get the id of the measure, as it is stored in the pivot formula
+     */
+    id: measure.id,
+    nameWithAggregator: fieldName + (aggregator ? `:${aggregator}` : ""),
```

Comment:
`nameWithAggregator` can be removed. It's only used for a `t-key` and it'll be wrong as soon as we have duplicates measure + aggregator